components/switch/test/switch-visibility.visual-diff.html

```diff
@@ -29,5 +30,79 @@
 		<d2l-switch-visibility id="on" on></d2l-switch-visibility>
 	</div>
 
+	<div class="visual-diff">
+		<d2l-switch-visibility id="off-with-conditions">
+			These are some conditions that must be met for the activity to be visible.
+			<ul>
+				<li> Condition 1 </li>
+				<li> Condition 2 </li>
+				<li> Condition 3 </li>
+			</ul>
+		</d2l-switch-visibility> 
+	</div>
+
+	<div class="visual-diff">
+		<d2l-switch-visibility id="on-with-conditions" on>
+			These are some conditions that must be met for the activity to be visible.
+			<ul>
+				<li> Condition 1 </li>
+				<li> Condition 2 </li>
+				<li> Condition 3 </li>
+			</ul>
+		</d2l-switch-visibility> 
+	</div>
+
+	<div class="visual-diff">
+		<d2l-switch-visibility id="on-with-conditions-text-position-start" on text-position="start">
+			These are some conditions that must be met for the activity to be visible.
+			<ul>
+				<li> Condition 1 </li>
+				<li> Condition 2 </li>
+				<li> Condition 3 </li>
+			</ul>
+		</d2l-switch-visibility> 
+	</div>
+
+	<div class="visual-diff">
+		<d2l-switch-visibility id="off-text-overridden" text="Label text has been overridden."></d2l-switch-visibility>
+	</div>
+
+	<div class="visual-diff">
+		<d2l-switch-visibility id="on-text-overridden" on text="Label text has been overridden."></d2l-switch-visibility>
+	</div>
+
+	<div class="visual-diff">
+		<d2l-switch-visibility id="off-with-conditions-text-overridden" text="Label text has been overridden.">
+			These are some conditions that must be met for the activity to be visible.
+			<ul>
+				<li> Condition 1 </li>
+				<li> Condition 2 </li>
+				<li> Condition 3 </li>
+			</ul>
+		</d2l-switch-visibility> 
+	</div>
+	
+	<div class="visual-diff">
```

Comment:
Since it's the usecase in LCS, perhaps also having a case for `text-position="hidden"` overriding `text` would be good as well.

Reply:
Yeah fair point. I was thinking, without considering the LCS usecase, that technically things shouldn't matter visually in terms of whether the visibility switch `has conditions` or not, whether the `text` is being overridden or not etc.

But yeah since it _is_ a use case, it would be good to have, for example, for regression checks.